devicetypes/sky-nie/min-smart-plug.src/min-smart-plug.groovy

```diff
@@ -85,20 +138,25 @@ private static def getCheckInterval() {
 def updated() {
     if (!isDuplicateCommand(state.lastUpdated, 5000)) {
         state.lastUpdated = new Date().time
-
         logDebug "updated()..."
         if (device.latestValue("checkInterval") != checkInterval) {
             sendEvent(name: "checkInterval", value: checkInterval, displayed: false)
         }
-
+        if (isButtonAvailable() &&  !childDevices) {
+            try {
+                def child = addChildButton()
+                child?.sendEvent(name: "checkInterval", value: checkInterval, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
```

Comment:
You don't have to define `checkInterval` for a child device. It should be sufficient to refresh child device's state in a parent DTH `refresh()` function. Additionally, if this is a separate button (battery powered ?) this device will probably be sleepy for most of the time.

Reply:
OK! I deleted it. The button is not a separate button, it's added only for the switch has parts of the button' attributes to set the paddle's dircetion.So we just to care the switch.